trieste/acquisition/function/greedy.py

```diff
@@ -16,17 +16,21 @@
 """
```

Comment:
name of the file is not descriptive enough - perhaps greedy_batch.py?

Reply:
sounds good, will do